applications/climatology/climatology.ipynb

```diff
@@ -0,0 +1,3533 @@
+{
```

Comment:
<p>This is a minor point, but I think it'd be useful to break this up a little bit. Something like:</p><p> </p>

<pre class="ql-syntax" spellcheck="false"># get data
filepath = DATASETS.fetch("CESM2_sst_data.nc")
ds = xr.open_dataset(filepath)

# calculate monthly anomaly
tos_monthly = ds.tos.groupby(ds.time.dt.month)
tos_clim = tos_monthly.mean(dim="time")
tos_anom = tos_monthly - tos_clim
tos_anom
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/71/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>